pkgs/applications/networking/droopy/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, buildPythonApplication }:
+
+with lib;
+
+buildPythonApplication rec {
```

Comment:
`stdenv.mkDerivation` with `wrapPython` in nativeBuildInputs should be enough here as the package doesn't `setup.py` and it also doesn't have any dependencies other than the default.